docs/package.md

```diff
@@ -0,0 +1,91 @@
+# The Coalton `(package ...)` Form
+
+In Coalton, the `package` form is used to define a package by
+specifying which dependencies to import, and which definitions to
+export.
+
+The form also establishes the current package: any definitions that
+follow `(package <name>)` will be contained within that package.
```

Comment:
maybe something like 

"... This replaces the explicit call of `(in-package <name>)` used in Lisp programs."